website/snippets/_cloud-environments-info.md

```diff
@@ -42,3 +42,48 @@ By default, all environments will use the default branch in your repository (usu
 - **Deployment:** determines the branch is cloned during job executions for each environment.
 
 For more info, check out this [FAQ page on this topic](/faqs/Environments/custom-branch-settings)!
+
+
+### Extended Attributes (Beta)
+
+:::important This feature is currently in beta
+
+Extended Attributes allow users to enter YAML attributes in their Environment settings. It's currently in beta and if you're interested in joining the beta, please contact us -- HOW CAN USES TRY THIS BETA? IF IT'S FEATURE FLAG, HOW DO THEY JOIN THE BETA? DO WE NEED TO ADD A FORM LINK? OR EMAIL?
+:::
+
+Extended Attributes is a feature that allows users to set a flexible [profiles.yml](/docs/core/connect-data-platform/profiles.yml) snippet in their dbt Cloud Environment settings. It provides users with more control over environments (both deployment and development) and extends how dbt Cloud connects to the data platform within a given environment.
+
+Extended Attributes is a text box extension at the environment level that overrides connection or environment credentials. It accepts any attributes accepted by a dbt adapter in its `profiles.yml`. 
+
+However, something to note is that it doesn't mask secret values. We recommend avoiding setting secret values to prevent visibility in the text box and logs.
+
+<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/extended-attributes.jpg" width="95%" title="Extended Attributes helps users add profiles.yml attributes to dbt Cloud Environment settings using a free form text box." /> <br />
+
+The **Extended Attributes** text box must contain valid characters only. Any invalid characters will result in an error message. Invalid characters include:
+
+- `/` (forward slash)
+- `\` (backslash)
+- `$` (dollar sign)
+- `!` (exclamation point)
+- `@` (at sign)
+- `#` (hash) (NEED CONFIRMATION)
```

Comment:
is this right?

Reply:
Nope, in my testing, we can accept all of those characters! Are you seeing something different? 

![Screenshot 2023-08-11 at 2 20 48 PM](https://github.com/dbt-labs/docs.getdbt.com/assets/24817191/fdc88625-1d8b-4144-8b21-b35e663d90d4)

Reply:
Additionally, the user can save these as strings - in which case, they will need to 'escape' any `\` by replacing it with `\\`

eg --- 
schema: "/\\$!@#+%"

will yield the same result as above

Reply:
hey @ericbaumann , gotcha! i dont have the ff but i got this info from the figma file in [this notion doc](https://www.notion.so/dbtlabs/Extended-Attributes-Landing-Page-7e806506e35b4cb8bce50d425c79d9d8#dacddb17f2cc4e2a833c42a518ce4cd0) which states the invalid characters AND also includes the password attribute


<img width="626" alt="Screenshot 2023-08-14 at 13 52 09" src="https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/ab836321-6693-4f73-b679-bebbb23b7632">
<img width="623" alt="Screenshot 2023-08-14 at 13 52 14" src="https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/5f9b0fe1-96bd-406a-a8b2-07027187a444">

Reply:
I would remove lines 62 to 72. Anything valid in YAML (the profile) is valid in the textbox.